frontend/src/app/pages/desktop.courseDetails.jsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+import './styles/desktop.courseDetails.scss';
+
+import CourseDetails from '../course/courseDetails';
+import CourseHeader from '../course/courseHeader';
+
+const CourseDetailsPage = (): React.Node => {
```

Comment:
The component should match the file name, in your case either rename the component as CourseDetails or rename the file into desktop.CourseDetailsPage